<ui version="4.0" >
 <author>José Manuel Díez Botella</author>
 <class>acumuladosmes</class>
 <widget class="QDialog" name="acumuladosmes" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Acumulados de cuenta por mes</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>9</number>
   </property>
   <item>
    <widget class="QLabel" name="titulotextLabel" >
     <property name="font" >
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>- -</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string/>
       </property>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>9</number>
        </property>
        <item>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="textLabel1" >
            <property name="text" >
             <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'DejaVu Sans'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans Serif'; font-size:9pt;">&lt;span style=" font-weight:600;">CUENTA&lt;/span>&lt;span style=" font-size:10pt; font-weight:600;">:&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="subcuentalineEdit" >
            <property name="maxLength" >
             <number>40</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buscapushButton" >
            <property name="minimumSize" >
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="icon" >
             <iconset resource="keme.qrc" >
              <normaloff>:/images/librok-55.png</normaloff>:/images/librok-55.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="anteriorpushButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Maximum" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>Subcuenta anterior</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="keme.qrc" >
         <normaloff>:/graf/1leftarrow.png</normaloff>:/graf/1leftarrow.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="posteriorpushButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Maximum" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>Subcuenta posterior</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="keme.qrc" >
         <normaloff>:/graf/1rightarrow.png</normaloff>:/graf/1rightarrow.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string/>
       </property>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>9</number>
        </property>
        <item>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="textLabel1_2" >
            <property name="text" >
             <string>&lt;b>EJERCICIO:&lt;/b></string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="ejerciciocomboBox" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tabla" >
     <property name="columnCount" >
      <number>5</number>
     </property>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar" >
     <property name="value" >
      <number>0</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="refrescarpushButton" >
       <property name="text" >
        <string>&amp;Refrescar</string>
       </property>
       <property name="icon" >
        <iconset resource="keme.qrc" >
         <normaloff>:/images/librok-54.png</normaloff>:/images/librok-54.png</iconset>
       </property>
       <property name="shortcut" >
        <string>Alt+R</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copiarpushButton" >
       <property name="text" >
        <string>&amp;Copiar</string>
       </property>
       <property name="icon" >
        <iconset resource="keme.qrc" >
         <normaloff>:/graf/Copy.png</normaloff>:/graf/Copy.png</iconset>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="salirpushButton" >
       <property name="text" >
        <string>&amp;Salir</string>
       </property>
       <property name="icon" >
        <iconset resource="keme.qrc" >
         <normaloff>:/images/librok-52.png</normaloff>:/images/librok-52.png</iconset>
       </property>
       <property name="shortcut" >
        <string>Alt+S</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>subcuentalineEdit</tabstop>
  <tabstop>ejerciciocomboBox</tabstop>
  <tabstop>refrescarpushButton</tabstop>
  <tabstop>copiarpushButton</tabstop>
  <tabstop>salirpushButton</tabstop>
  <tabstop>buscapushButton</tabstop>
 </tabstops>
 <resources>
  <include location="keme.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>salirpushButton</sender>
   <signal>clicked()</signal>
   <receiver>acumuladosmes</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
